### PR TITLE
doc: Update blog tutorial to updated syntax and instructions

### PR DESCRIPTION
**Pull request by HankG**
This PR updates the blog tutorial and the corresponding code to conform with current V/VLib syntax. Specifically:
* Changes project init to use `v init`
* Breaks up the `app` creation so that the `app.init_server()` call can be made in `main`
* Explicitly scopes `db` as `pub` to fix compile time error
* Adds ORM decorators on `id` to eliminate runtime `datatype mismatch` error
* Moves the file creation version to end since in memory works, and gets rid of initialization file since code defined tables
* Added explicit vweb decorators for endpoints
* Added explicit `new` method to host `new.html` form file.

### CODE REVIEW

File: tutorials/building_a_simple_web_blog_with_vweb/README.md [hunk 1]
```diff
@@ -391,14 +394,35 @@ in V is very simple:
...
+	app.db = sqlite.connect('blog.db') or { panic(err) }
+	app.db.create_table('Article', [
+		'id integer primary key',
+		"title text default ''",
+		"text text default ''",
+	])
+}
+```
+
+As we can see it attempts to connect to a file in the active pathcalled `blog.db`.
```
Comment (JalonSolov) [inline tutorials/building_a_simple_web_blog_with_vweb/README.md:422]: attemps to open a database file in the current path named `blog.db`

File: tutorials/building_a_simple_web_blog_with_vweb/README.md [hunk 2]
```diff
@@ -372,12 +363,24 @@ pub fn (mut app App) new_article() vweb.Result {
...
 We need to update `index.html` to add a link to the "new article" page:
 
 ```html
 <a href='/new'>New article</a>
 ```
 
+Next we need to add the HTML endpoint to our code like we did with `index.html`:
+
+```v ignore
+pub fn (mut app App) new() vweb.Result {
```
Comment (JalonSolov) [inline tutorials/building_a_simple_web_blog_with_vweb/README.md:378]: New to add `[/new]` route attribute.  A change is going in that will make you have to have explicit route attributes for the endpoint to be exposed.

File: tutorials/building_a_simple_web_blog_with_vweb/README.md [hunk 3]
```diff
@@ -262,11 +250,14 @@ pub fn (app &App) find_all_articles() []Article {
...
+Notice that the `Article` structure conforms to the same structure and naming as 
+the database table in the creation SQL statement. Also we need to add ORM decorators
+to our primary key to let it know that it is the primary key and it should auto-increment
+
 Let's fetch the articles in the `index()` action:
 
 ```v ignore
-// blog.v
-pub fn (app &App) index() vweb.Result {
+pub fn (app App) index() vweb.Result {
```
Comment (JalonSolov) [inline tutorials/building_a_simple_web_blog_with_vweb/README.md:260]: New to add `[/index]` route attribute. A change is going in that will make you have to have explicit route attributes for the endpoint to be exposed.

File: tutorials/building_a_simple_web_blog_with_vweb/README.md (hunk 1 above)
Comment (HankG) [reply tutorials/building_a_simple_web_blog_with_vweb/README.md]: updated

File: tutorials/building_a_simple_web_blog_with_vweb/README.md (hunk 2 above)
Comment (HankG) [reply tutorials/building_a_simple_web_blog_with_vweb/README.md:380]: Updated in README.md and the source code.

File: tutorials/building_a_simple_web_blog_with_vweb/README.md (hunk 3 above)
Comment (HankG) [reply tutorials/building_a_simple_web_blog_with_vweb/README.md:261]: Updated in README.md and the source code.

File: tutorials/building_a_simple_web_blog_with_vweb/README.md [hunk 4]
```diff
@@ -391,14 +396,35 @@ in V is very simple:
...
+	app.db = sqlite.connect('blog.db') or { panic(err) }
+	app.db.create_table('Article', [
+		'id integer primary key',
+		"title text default ''",
+		"text text default ''",
+	])
+}
+```
+
+As we can see it attempts to connect to a file in the active path named `blog.db`.
```
Comment (JalonSolov) [inline tutorials/building_a_simple_web_blog_with_vweb/README.md:424]: "connect to a file in the active path" -> "open a file in the current directory"

**Review by JalonSolov — APPROVED** [review]

File: tutorials/building_a_simple_web_blog_with_vweb/README.md (hunk 4 above)
Comment (HankG) [reply tutorials/building_a_simple_web_blog_with_vweb/README.md]: Updated

**Comment by HankG** [comment]
Since this was still here I decided to make another change. The table creation commands are now using the ORM syntax not the raw SQL syntax.

**Comment by spytheman** [comment]
The latest CI failure is unrelated.
Good work!

**Pull request merged**